### ISSUE TITLE
Bug of parse_old_attributes

### ISSUE THREAD

**Issue by creasty** [k0kubun/hamlit#7]
Hi, thanks for your great work, I'm seriously considering replacing `haml-rails` with `hamlit` in my project, but I'm in trouble with a bug of `parse_old_attributes`.

### Accepted and correct

```ruby
parse_old_attributes '{ data: { value: "#{I18n.t "foo"}"}}'
#=> {"data"=>{"value"=>"\"\#{I18n.t \"foo\"}\""}}
```

### Accepted but incorrect

```ruby
parse_old_attributes '{ value: "#{I18n.t "foo", count: 1}" }'
#=> {"value"=>"\"\#{I18n.t \"foo\", count: 1", "{"=>"\""}

parse_old_attributes '{ data: { value: "#{I18n.t "foo", count: 1}" } }'
#=> {"data"=>{"value"=>"\"\#{I18n.t \"foo\", count: 1", "{"=>"\""}}
```

### Rejected

Watch braces at the end of buffer

```ruby
parse_old_attributes '{ data: { value: "#{I18n.t "foo", count: 1}"}}'
# raises #<Hamlit::SyntaxError: Hamlit::SyntaxError>
```

**Comment by k0kubun** [comment]
Thank you for your attention to this project and I fixed the issue in ff53e5a.
Released as v0.3.2 :gem:

**Comment by creasty** [comment]
Thank you for your fast response!
